test/common/README.md

```diff
@@ -184,12 +184,13 @@ Array of IPV6 hosts.
 
 ### mustCall([fn][, expected])
 * fn [&lt;Function>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function)
-* expected [&lt;Number>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type) default = 1
+* expected [&lt;Number>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type) default = 1 |
+ [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type) '+'
 * return [&lt;Function>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function)
 
-Returns a function that calls `fn`. If the returned function has not been called
-exactly `expected` number of times when the test is complete, then the test will
-fail.
+Returns a function that calls `fn`. If the returned function has not been
+called exactly `expected` number of times, or at least once if
+`expected === '+'`, when the test is complete, then the test will fail.
```

Comment:
Nit: This documentation change results in a confusing run-on sentence:

> If the returned function has not been called exactly `expected` number of times, or at least once if
`expected === '+'`, when the test is complete, then the test will fail.

Would be better as something like:

> If the returned function has not been called exactly `expected` number of times when the test is complete, then the test will fail. If `expected is `'+'`, then the test must run at least once.